isis/src/clipper/apps/jitterfit/jitterfit.xml

```diff
@@ -2,37 +2,89 @@
   <brief>
-    Registers lines from a normal image with lines from the check image.
+    Characterize jitter for rolling shutter images.
   </brief>
   <description>
     <p>
-      This program co-regsisters lines in the main cube (FROM) with the corrisponding line in
-      the checkline cube (FROM2).
-      The sensor line used to collect the checkline is used to center the registration around
-      the same area in the main cube.
+      This program calculates a two sets of polynomial coefficients that characterize the jitter in an
+      image acquired by a rolling shutter instrument. One polynomial for the line direction and on
+      for the sample direction. The ISIS camera model for the instrument can
+      then use the coefficients to adjust for the time dependent jitter.
     </p>
     <p>
-      The main input cube (FROM) is expected to be a cube from a rolling shutter instrument.
-      The checkline cube (FROM2) is expected to contain lines of data read from the same rolling shutter
-      instrument, but read at different times than the coorsponding line in the main cube. The check 
-      cube must also have an associated table containing the sensor line number and time the line was read. 
-      The algorithum co-registers these individual checklines to the main cube and calculates an offset
-      in the line and sample directions. It then fits an Nth order polynomial through the offsets and time to
-      characterize the jitter. The coefficients are then written to the cube labels so the rolling shutter
-      camera model can adjust the look direction of each pixel in the model's detector map.
+      The coefficients are
+      calculated by coregistering lines in the cube with each line in the check line cube, where each
+      check line was acquired durring the readout of the main cube using the
+      same detector, but at a different time. The registration produces sample and line offsets
+      which are fed into a least squares solution to determin the coeficients of an Nth degree
+      polynomial in time. The coefficients are written to the cube label for use by the ISIS camera model.
+    </p>
+    <p>
+      The main cube (FROM) is expected to be a full frame image collected by a rolling shutter sensor.
+      The cube must have two attached tables: 1) A table with a record for each line in the cube where
+      the first column contains the sensor line number used to collect the corresponding image line,
+      and the second column contains the associated exposure time (normalized -1 to 1). 2) A table
+      with one record for each check line where the first column is the sensor line number used to
+      collect the corresponding check line and the second column contains the time the line was
+      exposed normalized using the same minimum and maximum as the main image times were
```

Comment:
On the normalization: I think what you wrote is clear, but does it have enough detail for someone external to do this normalization if they needed to to create input data for `jitterfit` _and_ do we expect this to ever happen, or do we expect to handle this for the user (if so, how?)